frontend/views/pages/Page.vue

```diff
@@ -85,29 +91,46 @@ $pagePaddingDesktop: 75px;
 
 .p-main {
   grid-area: p-main;
-  overflow: auto;
+  padding-left: $spacer;
```

Comment:
Maybe this should also have `padding-top` to avoid cutting the shadow of the card that is noticeable on the card corner:

Without padding:
![image](https://user-images.githubusercontent.com/14869087/64180861-69928080-ce5d-11e9-9947-7fc0f51b6229.png)

With some padding:
![image](https://user-images.githubusercontent.com/14869087/64180914-7a42f680-ce5d-11e9-83ab-c672a1910059.png)

Reply:
Overflow visible will work better ;)